tests/ntp/test_ntp.py

```diff
@@ -11,9 +12,21 @@
     pytest.mark.device_type('vs')
 ]
 
+TIME_FORWARD = 3600
+
+def config_long_jump(duthost, enable=False):
+    """change ntpd option to enable or disable long jump"""
+    if enable:
+        logger.info("enable ntp long jump")
+        regex = "s/NTPD_OPTS='-x'/NTPD_OPTS='-g'/"
+    else:
+        logger.info("disable ntp long jump")
+        regex = "s/NTPD_OPTS='-g'/NTPD_OPTS='-x'/"
+
+    duthost.command("sed -i %s /etc/default/ntp" % regex)
```

Comment:
You need to restart ntp service to complete the configuration.

Reply:
Fixed, and the restarts next it were also removed